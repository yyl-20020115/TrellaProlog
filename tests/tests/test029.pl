:-initialization(main).

foo(a,b,c).
foo(a,b,d).
foo(b,c,e).
foo(b,c,f).
foo(c,c,g).
foo(d,e,g).

main :- between(1,10,_),bagof(C, foo(_,_,C), Cs), write(Cs), nl, fail.
main.
