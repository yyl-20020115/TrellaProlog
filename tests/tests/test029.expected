"cd"
"ef"
"g"
"g"
"cd"
"ef"
"g"
"g"
"cd"
"ef"
"g"
"g"
"cd"
"ef"
"g"
"g"
"cd"
"ef"
"g"
"g"
"cd"
"ef"
"g"
"g"
"cd"
"ef"
"g"
"g"
"cd"
"ef"
"g"
"g"
"cd"
"ef"
"g"
"g"
"cd"
"ef"
"g"
"g"
